keyboards/adelheid/info.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
This produces 
![image](https://user-images.githubusercontent.com/16520359/87781200-27312280-c828-11ea-9fb7-62108a5e20d7.png)

You can preview the file on config.qmk.fm with `ctrl+shift+i`

Reply:
Thanks! I see where I went wrong, fixed it.

Reply:
Looks much better, order _seems_ ok with fast mode too.